OutOfSchool/OutOfSchool.BusinessLogic/Services/UserService.cs

```diff
@@ -48,23 +48,20 @@ public async Task<IEnumerable<ShortUserDto>> GetAll()
         return users.Select(user => mapper.Map<ShortUserDto>(user)).ToList();
     }
 
-    // TODO: use repository.GetById() method
     public async Task<ShortUserDto> GetById(string id)
     {
-        logger.LogInformation($"Getting User by Id started. Looking Id = {id}.");
+        logger.LogInformation("Getting User by Id started. Looking Id = {Id}.", id);
 
-        Expression<Func<User, bool>> filter = p => p.Id == id;
-
-        var users = await repository.GetByFilter(filter).ConfigureAwait(false);
+        var user = await repository.GetById(id).ConfigureAwait(false);
```

Comment:
(Performance) Interesting change of code. 
I presume it changes nothing in logic, but as you've touched this place I  have to check `.AsNoTracking()` moment (as retrieved `user` is then mapped to `Dto` we seems to need no tracking, right?). 

Could you confirm that `repository.GetById()` doesn't use tracking?
If not - could you rewrite it to version without tracking?